kohttp/src/main/kotlin/io/github/rybalkinsd/kohttp/configuration/Config.kt

```diff
@@ -18,6 +20,11 @@ internal data class ClientConfig(
     val dispatcher: DispatcherConfig = DispatcherConfig()
 )
 
+data class SslConfig(
```

Comment:
nit: Looks like we don't benefit from making this class `data`. 

The other classes in this file are `internal` so it will be possible for us to change them any time in the future, however, `SslConfig` will be available for the end user together with destructuring methods ( `componentN()` ).

`componentN()` methods destructs data classes based only on field position like ` for ((k, v) in map)`. Could be a fragile construction

Reply:
Ah good point, I didn't know that. I removed the data keyword from the SslConfig class